gb.yaml

```diff
@@ -507,6 +513,21 @@ tests:
       regions: ["gb"]
     expect:
       name: "Platinum Jubilee"
+  - given:
+      date: '2022-09-18'
```

Comment:
Should this be the 19th instead?
```suggestion
      date: '2022-09-19'
```

Reply:
Yes, you are right... Fixed!

Actually, that means (checks...) ... that `make test` doesn't run the `expect` sections?

Reply:
No, the tests are run within a separate repo (https://github.com/holidays/holidays ) which imports the definitions and the tests.